src/features/wallet/send/SendForm.tsx

```diff
@@ -127,17 +137,31 @@ const SendForm = ({
         disabled={!isValid}
       />
       {shouldShowFee && <FeeFooter fee={fee} />}
+      {warning && <NoticeFooter text={warning} color="redMain" />}
```

Comment:
Could potentially show a warning/notice when Secure Mode is enabled with an address that says something like "Payments can only be sent to the address above" but that would (a) compete with the fee footer and (b) may not be necessary if you know Secure Mode is already enabled (and know how it works)